### ISSUE TITLE
Decide strategy to add icons to the navigation items

### ISSUE THREAD

**Issue by ckrina** [Lullabot/navigation#17]

**Comment by mherchel** [comment]
Lets just inline the SVGs

**Comment by lauriii** [comment]
That's fine for now.  For core, I'm wondering if that's the best approach given that themes may have to override the icons and new menu links need to be able to provide icons

**Comment by ckrina** [comment]
Yeah, I think this strategy needs to be discussed a little bit even if it's not just for the initial prototype but something we implement when we jump to twig.

We could be talking about:
- SVGs as background images, either in the element or sub-element. Implications of this is not being able to change the icon colors with an accent color.
- Inline SVGs. How do we implement this in the next phase in twig (when moving this to contrib or patch to core)?
- Libraries. Would improve Editor experience exponentially, but open the Pandora box to a lot of things. Copyright would be "easy" once we agree on something, but what about getting rid of it when it's not maintained anymore?
